src/databricks/sql/utils.py

```diff
@@ -144,9 +144,9 @@ def escape_string(self, item):
         if isinstance(item, bytes):
             item = item.decode("utf-8")
         # This is good enough when backslashes are literal, newlines are just followed, and the way
-        # to escape a single quote is to put two single quotes.
+        # to escape a single quote is to precede it with 2 backslashes.
         # (i.e. only special character is single quote)
-        return "'{}'".format(item.replace("'", "''"))
+        return "'{}'".format(item.replace("'", "\\'"))
```

Comment:
i know that per @susodapop this will be fixed elsewhere, but want to point out that the backslash character itself also needs to be escaped

i think the correct replacement would be: `item.replace('\\','\\\\' ).replace("'", "\\'")`